crates/polars-plan/src/dsl/python_udf.rs

```diff
@@ -298,3 +334,17 @@ impl Expr {
         }
     }
 }
+
+/// Get the minor Python version from the `sys` module.
+fn get_python_minor_version() -> u8 {
```

Comment:
Can we cache this behind a lazylock?

Reply:
Yes, done!

Unfortunately, now I can no longer patch the `sys.version_info` from the Python side, so we can no longer really test this. If you have an idea how to test it, let me know. Otherwise, we'll just have to trust the code :grimacing: